setup.py

```diff
@@ -12,12 +14,13 @@ def run(self, *args, **kwargs):
 
 setup(
     name="Python-Calculator",
-    version="0.2",
+    version=datetime.utcnow().isoformat(),
```

Comment:
How will you make the release from such version?
It should be like "0.2-SNAPSHOT", where you substitute "SNAPSHOT" substring in job runtime, and during release just reove thist tail.

Reply:
Fixed